.github/workflows/base-package-tests.yaml

```diff
@@ -28,3 +33,29 @@ jobs:
 
       - name: Test ${{ matrix.type }} package
         run: ./scripts/package-tests/package-tests.sh ./otelcol*-SNAPSHOT-*_linux_amd64.${{ matrix.type }} ${{ inputs.distribution }}
+
+  create-issue:
+    name: Create GitHub Issue
+    runs-on: ubuntu-22.04
+    needs: [package-tests]
+    if: failure() && inputs.issue-on-failure == 'true'
+    steps:
+      - name: Formulate issue
+        id: formulate_issue
+        run: |
+          # create a markdown file that contains details about the failure
+          echo "The nightly linux package tests failed in the following GitHub actions run." > nightly-test-failure.md
+          echo "* Link to run: $GITHUB_SERVER_URL/$GITHUB_REPOSITORY/actions/runs/$GITHUB_RUN_ID" >> nightly-test-failure.md
+          echo "* Triggered by: Nightly scheduled build" >> nightly-test-failure.md
+          echo "" >> nightly-test-failure.md
+          echo "Note: This issue was auto-generated from [base-package-tests.yaml](https://github.com/open-telemetry/opentelemetry-collector-releases/blob/main/.github/workflows/base-package-tests.yaml)" >> nightly-test-failure.md
+
+      - name: Create issue on failure
+        env:
+          GITHUB_TOKEN: ${{ secrets.GH_PAT }}
+        run: |
+          gh issue create \
+            --repo open-telemetry/opentelemetry-collector-contrib \
```

Comment:
question: why always in contrib?

Reply:
that was requested by @mx-psi in the original comment

Reply:
I guess not all changes would be things to fix in contrib, but my bet is that most of them would be, thus why I suggested putting them in contrib. Since this is easy to change and does not have an impact on end users, we can start with this and see how it goes